src/main/java/org/sagebionetworks/bridge/config/PropertiesConfig.java

```diff
@@ -125,6 +133,24 @@ public PropertiesConfig(final Path configTemplate, final Path localConfig,
         delimiter = Pattern.compile(delimiterRegex);
     }
 
+    public PropertiesConfig(final String configTemplate, final Path localConfig,
+                            final String delimiterRegex) throws IOException {
```

Comment:
Can you combine this constructor with the other one so there's no duplicate code?